skll/metrics.py

```diff
@@ -215,6 +227,95 @@ def f1_score_least_frequent(y_true, y_pred):
     return f1_score(y_true, y_pred, average=None)[least_frequent]
 
 
+def register_custom_metric(custom_metric_path, custom_metric_name):
+    """
+    Import, load, and register the custom metric function from the given path.
+
+    Parameters
+    ----------
+    custom_metric_path : str
+        The path to a custom metric.
+    custom_metric_name : str
+        The name of the custom metric function to load.
+        This function must take only two array-like
```

Comment:
Doesn't the documentation in `custom_metrics.rst` say that there are also three optional keyword arguments?

Reply:
For the arbitrary custom metric function that _you_ are writing, not for this function :)

Reply:
ah, of course